terraform/pagerduty/kms.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_kms_key" "pagerduty" {
+  tags = local.tags
+}
+
+resource "aws_kms_alias" "pagerduty" {
+  target_key_id = aws_kms_key.pagerduty.id
```

Comment:
Do you want to add in a name here to be consistent with the other kms alias resources?

Reply:
I do! Good spot!